### ISSUE TITLE
Feature: choose custom title text

### ISSUE THREAD

**Issue by noraj** [FHPythonUtils/AnsiToImg#7]
## Feature

- [x] I have read the comment above and have completed each step
- [x] I have filled in each heading below

---

- Be able to remove the titlebar watermark (`AnsiToImg (courtesy of Rich)`)
- Choose custom titlebar text

### Describe the solution you'd like

add a `--titlebar <text>` option that override the hardcoded watermark

**Comment by FredHappyface** [comment]
Hey thanks for the request. Seems reasonable tbh. The watermark was intended as a way to credit rich as it really does the heavy lifting here. Originally each rendering function was implemented entirely by this lib, however, I chose to stand on the shoulders of giants in more recent versions. With this in mind, providing credit in this way seemed appropriate.

I'll add this to me Todo. Can't provide a good ETA as I work on this in my spare time
